server/store/config.go

```diff
@@ -0,0 +1,56 @@
+package store
+
+import (
+	"bytes"
+	"encoding/json"
+
+	"github.com/pkg/errors"
+)
+
+//Config configuration for the plugin
+type Config struct {
+	Org     string
+	Project string
+}
+
+const (
+	configStoreSuffix = "_circleci_config"
```

Comment:
Nit pick: I think it should be
```suggestion
	configStoreSuffix = "_user_config"
```
as it isn't directly relative to CircleCI

Reply:
It is, right.. This is the plugin config.. We might add more to this in future . For example. The circleCi server url in case we need to support standalone version of circle later..